synapse/handlers/presence.py

```diff
@@ -1962,6 +2042,7 @@ def handle_update(
     # If the users are ours then we want to set up a bunch of timers
     # to time things out.
     if is_mine:
+        # TODO Maybe don't do this if currently active?
```

Comment:
Fortunately, these aren't 'active timers', just a timestamp placeholder checked over every 5 seconds for new stuff. I thought it was contributing to the overhead of presence, it's not. At least not directly. I think it should be fine to keep as-is.